File: LV1/부족한 금액 계산하기.js
function solution(price, money, count) {
  let answer = 0;
  let howmuch = 0;

  for (let i = 1; i <= count; i++) {
    howmuch += price * i;
  }
  howmuch - money < 0 ? (answer = 0) : (answer = howmuch - money);

  return answer;
}

File: LV1/부족한 금액 계산하기.java
class Solution {
    public long solution(int price, int money, int count) {
        long answer = 0
        long howmuch = 0;
        for (int i = 1; i <= count; i++) {
            howmuch += price * i;
        }
        return howmuch - money < 0 ? 0 : howmuch - money;
    }
}